style_guides/js_style_guide.md

```diff
@@ -344,92 +476,69 @@ if (a == '') {
 
 ## Only use ternary operators for small, simple code
 
-And **never** use multiple ternaries together
-
-*Right:*
+And *never* use multiple ternaries together
 
 ```js
-var foo = (a === b) ? 1 : 2;
-```
-
-*Wrong:*
+// good
+const foo = (a === b) ? 1 : 2;
```

Comment:
I see. So you're saying this demonstration shows that the circumstance where a ternary is appropriate, is where there is only 1 ternary, right? If so then I think we can update the comment to make that clear (because it wasn't clear to me 😄):

```
// good, a situation where only 1 ternary is needed
```